lib/git-subrepo

```diff
@@ -531,7 +539,10 @@ subrepo:init() {
 
   o "Commit new subrepo to the '$original_head_branch' branch."
   subrepo_commit_ref=$original_head_commit
-  RUN git commit -m "$(get-commit-message)"
+  local no_verify_flag=' --no-verify'
+  $verify_wanted && no_verify_flag=''
+  # shellcheck disable=2086
+  RUN git commit$no_verify_flag -m "$(get-commit-message)"
```

Comment:
To prevent disabling 2086 this should use Parameter Expansion https://www.gnu.org/software/bash/manual/html_node/Shell-Parameter-Expansion.html#Shell-Parameter-Expansion-1